models/intel/text-recognition-0016/accuracy-check.yml

```diff
@@ -0,0 +1,108 @@
+evaluations:
+  - name: text-recognition-0016
+    module: custom_evaluators.custom_text_recognition_evaluator.TextRecognitionWithAttentionEvaluator
+    module_config:
+      model_type: SequentialTextRecognitionModel
+      lowercase: true
+      network_info:
+        max_seq_len: "24"
```

Comment:
after merging https://github.com/openvinotoolkit/open_model_zoo/pull/2880 config should be updated
cc @Anna-Mironova